lib/utils/form-builder/validate.ts

```diff
@@ -22,7 +22,7 @@ const getErrorMessageTranslationString = (error: ValidationError) => {
   };
 };
 
-export const validateTemplate = (data: FormRecord) => {
```

Comment:
In Preview the type used for an uploaded JSON file is FormProperties. I'm pretty sure that's what was intended here.